### PR TITLE
feature (refs T33519): add agency main email address to service output

### PR DESCRIPTION
**Pull request by jalalifardemos**
**Ticket:** https://yaits.demos-deutschland.de/T33519 

Description: DemosPlanProcedureContrlooer::getProcedureWithPhaseNames() must return string but null.

### How to review/test
test in browser

Delete the checkbox if it doesn't apply/isn't necessary.

- [ ] Tests updated/created
- [ ] Update documentation
- [x] Link all relevant tickets
- [x] Move the tickets on the board accordingly

### CODE REVIEW

**Review by bakridemos — CHANGES_REQUESTED** [review]
i don't really think the adjustements in the addPhaseNames method are necessary, it's not the right context and i don't really understand why you need the internalPhases here ?  i think we could fix this in the Controller with a simple Check, if the agencyMainEmailAddress is null or not and try to pass an empty string if its null.

**Review by bakridemos — COMMENTED** [review]
it's look good to me, but since we did this together i suggest that you ll ask someone else to review this pr.

File: demosplan/DemosPlanCoreBundle/Controller/Procedure/DemosPlanProcedureController.php [hunk 1]
```diff
@@ -2143,7 +2143,8 @@ public function administrationMemberListAction(
 
         $emailTextAdded = '';
         if ($this->permissions->hasPermission('feature_email_invitable_institution_additional_invitation_text')) {
-            $emailTextAdded = $this->generateAdditionalInvitationEmailText($publicAffairsAgents, $organization, $procedureAsArray['agencyMainEmailAddress']);
+            $agencyMainEmailAddress = isset($procedureAsArray['agencyMainEmailAddress']) && null !== $procedureAsArray['agencyMainEmailAddress'] ? $procedureAsArray['agencyMainEmailAddress'] : '';
+            $emailTextAdded = $this->generateAdditionalInvitationEmailText($publicAffairsAgents, $organization, $agencyMainEmailAddress);
```
Comment (mussbach) [inline demosplan/DemosPlanCoreBundle/Controller/Procedure/DemosPlanProcedureController.php:2147]:
```suggestion
            $emailTextAdded = $this->generateAdditionalInvitationEmailText($publicAffairsAgents, $organization, $procedureAsArray['agencyMainEmailAddress'] ?? '');
```

**Review by mussbach — COMMENTED** [review]
This can be simplified

File: demosplan/DemosPlanCoreBundle/Controller/Procedure/DemosPlanProcedureController.php (hunk 1 above)
Comment (jalalifardemos) [reply demosplan/DemosPlanCoreBundle/Controller/Procedure/DemosPlanProcedureController.php]: done in [cb1a76c](https://github.com/demos-europe/demosplan-core/pull/1645/commits/cb1a76c566ac32de04160e486908541c32a6ae84)

**Review by mussbach — APPROVED** [review]
lgtm, code review only

**Pull request merged**